admin-console/src/main/webapp/src/components/SystemBehaviorGroups/CreateEditBehaviorGroupModal.tsx

```diff
@@ -0,0 +1,85 @@
+import { ActionGroup, Button, Form, FormGroup,
+    FormSelect,
+    FormSelectOption,
+    HelperText, HelperTextItem, Modal, ModalVariant, TextInput } from '@patternfly/react-core';
+import React from 'react';
+
+import { BehaviorGroup } from '../../types/Notifications';
+
+interface CreateEditModalProps {
+    isEdit: boolean;
+    showModal: boolean;
+    isOpen: boolean;
+    initialSystemBehaviorGroup?: Partial<BehaviorGroup>;
+    isLoading: boolean;
+    onClose: () => void;
+    onSubmit: (systemBehaviorGroup: Partial<BehaviorGroup>) => void;
+}
+
+export const CreateEditBehaviorGroupModal: React.FunctionComponent<CreateEditModalProps> = (props) => {
+
+    const [ systemBehaviorGroup, setSystemBehaviorGroup ] = React.useState<Partial<BehaviorGroup>>(props.initialSystemBehaviorGroup ?? {});
+
+    const actionOption = [
+        <FormSelectOption key={ 0 } isPlaceholder label='Choose an action' />,
+        <FormSelectOption key={ systemBehaviorGroup?.id } label='Send an email to Users: All' value={ systemBehaviorGroup?.id } />,
+        <FormSelectOption key={ systemBehaviorGroup?.id } label='Send an email to Users: Admins' value={ systemBehaviorGroup?.id } />
```

Comment:
Should not reuse keys for the same array. This key uniquely identifies **each** element in the array.
Also - I'm unsure what the value means here, but i guess it should have the intent of the action i.e.

```suggestion
        <FormSelectOption key="choose action" isPlaceholder label='Choose an action' />,
        <FormSelectOption key="email-all" label='Send an email to Users: All' value="email-all" />,
        <FormSelectOption key="email-admin" label='Send an email to Users: Admins' value="email-admin" />
```